src/App.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'
 import './App.css';
-import Header from './components/header/header'
+import Header from './components/header/header';
+import TakeSurvey from './components/home/TakeSurvey';
+
 
 function App() {
   return (
     <div>
       <Router>
       <Switch>
       <Route path="/" exact component={Header}/>
+      <Route path="/takesurvey" exact component={TakeSurvey}/>
```

Comment:
Here in this route! You have to do slight changes! 
1) Take a checkout from the master branch 
2) Instead of this header component you will find a home component!
3) And once you will open this home component, call this take survey component, in that home component.

rest everything is fine